karma.conf.js

```diff
@@ -81,7 +81,6 @@ module.exports = function(karma) {
 			plugins: [
 				json(),
 				resolve(),
-				babel({exclude: 'node_modules/**'}), // use babel since we have ES proposal features
```

Comment:
you can remove the import at the top of the file now

Reply:
and then probably the dependency from package.json too

Reply:
Its still used for UMD build and the "sources" build in tests. Just not deeded for the `/test/**.js`